modules/apps/asset/asset-auto-tagger-google-cloud-natural-language-impl/src/test/java/com/liferay/asset/auto/tagger/google/cloud/natural/language/internal/GCloudNaturalLanguageDocumentAssetAutoTaggerImplTest.java

```diff
@@ -48,11 +48,11 @@ public class GCloudNaturalLanguageDocumentAssetAutoTaggerImplTest {
 
 	@BeforeClass
 	public static void setUpClass() {
-		_gCloudNaturalLanguageDocumentAssetAutoTaggerImpl =
-			new GCloudNaturalLanguageDocumentAssetAutoTaggerImpl();
+		_gCloudNaturalLanguageDocumentAssetAutoTagger =
```

Comment:
@GabrielHenriquedev, please rename class to `GCloudNaturalLanguageDocumentAssetAutoTaggerTest`. Thx